openapi.yaml

```diff
@@ -3524,10 +3536,13 @@ components:
           - $ref: '#/components/schemas/ReleaseMetricID'
         name:
           $ref: '#/components/schemas/GoalTemplateName'
+        repositories:
+          $ref: '#/components/schemas/GoalRepositories'
       required:
       - id
       - name
       - metric
+      - repositories
```

Comment:
It shouldn't be required.

Reply:
You mean that the API is not required to include `repositories` in the response ?

Reply:
Yep

Reply:
fixed